README.md

```diff
@@ -236,35 +236,35 @@ assertProgressIsMax(R.id.seek_bar)
 
 ```
 
-### Barista custom assert
+### Custom assertions
 
-Do your own asserts on given view, no more custom *Matcher*
+If you have a special case not covered by the given assertions API, we encourage you to assert these special cases with our custom assertions API. It's a convenient way to replace plain `Matcher`s with complex assertions. With Barista, you can match any kind of view by knowing its type and passing its `viewId`, `text`, or a common `Matcher<View>`. Once you matched it, you will be able to assert all its properties without adding any complex `Matcher` to your project.
```

Comment:
`common` or `custom` Matcher?

Reply:
Mmmmm you can also use a common `withId` or those given by Espresso... but you are right. I think I could just avoid the word and it will work better 💃